.ember-cli

```diff
@@ -5,5 +5,11 @@
 
     Setting `disableAnalytics` to true will prevent any data from being sent.
   */
-  "disableAnalytics": true
+  "disableAnalytics": false,
```

Comment:
I had never heard about that, now I am curious to know what kind of data we're talking about.

Reply:
https://github.com/ember-cli/ember-cli/blob/master/docs/analytics.md

LGTM, + it aligns with other addons in https://github.com/DazzlingFugu